jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixServiceInitListener.java

```diff
@@ -129,6 +130,7 @@ protected void onSessionInitEvent(SessionInitEvent event) {
         }
         event.getSession().setErrorHandler(uiExceptionHandlers);
         event.getSession().setAttribute(BackgroundTaskManager.class, new BackgroundTaskManager());
+        event.getSession().setAttribute(UiEventsManager.class, new UiEventsManager());
```

Comment:
Some comment required, describing why we store something in session attributes

Reply:
Added a comment